### PR TITLE
Popovers and small buttons!

### PR DESCRIPTION
**Pull request by JPLachance**
+ Add the mod-small for buttons
+ Add the popover with his documentation
* Replace some components in the documentation
screen shot 2016-09-19 at 3 47 12 pm
screen shot 2016-09-19 at 4 08 26 pm

### CODE REVIEW

File: scss/components/popover.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.popover {
+  min-width: $popover-min-width;
+
+  background-color: $pure-white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.4);
```
Comment (pochretien) [inline scss/components/popover.scss]: u can use rgba($pure-white, 0.4) i think

File: scss/components/popover.scss [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  border: 1px solid $greyish-dark-blue;
+  border-radius: 2px;
```
Comment (pochretien) [inline scss/components/popover.scss]: use the variable

**Comment by pochretien** [comment]
:gg:

File: scss/components/popover.scss (hunk 1 above)
Comment (GermainBergeron) [inline scss/components/popover.scss]: It's black, not white ;)

Comment (JPLachance) [inline scss/components/popover.scss]: Done!

File: scss/components/popover.scss (hunk 2 above)
Comment (JPLachance) [inline scss/components/popover.scss]: Done!

**Pull request merged**